OpenRA.Mods.Common/Traits/Buildings/Refinery.cs

```diff
@@ -44,6 +44,9 @@ public class RefineryInfo : TraitInfo, Requires<WithSpriteBodyInfo>, IAcceptReso
 		[Desc("Discard resources once silo capacity has been reached.")]
 		public readonly bool DiscardExcessResources = false;
 
+		[Desc("Value of each accepted resource type.", "Dictionary of [resource type]: [value per unit].")]
+		public readonly Dictionary<string, int> ResourceValues = new Dictionary<string, int>();
```

Comment:
This is going to conflict with the logical next step, which is to convert resource storage from money to resources.

This should instead move to `PlayerResources`.